### PR TITLE
add K8s 1.25 windows presubmit and release job configs

### PR DESCRIPTION
**Pull request by nawazkh**
- Adds K8s:1.25 pre-submit sig-windows jobs configuration
- Adds K8s:1.25 sig-windows jobs configuration

TODOS:
- [x] ~update [windows-testing](https://github.com/kubernetes-sigs/windows-testing) repo with K8s:1.25 containerd json. Referring to [K8s:1.24 containerd json file](https://github.com/kubernetes-sigs/windows-testing/blob/master/job-templates/kubernetes_containerd_1_24_csi_enabled.json)~. Not applicable any more since we got rid of aks-engine related jobs.
- [x] ~Push kubekins-e2e:v20220812-2d111f2c30-1.25 image.~ Not applicable. Checked for its existence
- [x] Squash Commits

/sig windows

### CODE REVIEW

File: config/jobs/kubernetes-sigs/sig-windows/release-1.25-windows-presubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,217 @@
+# generated by ./config/jobs/kubernetes-sigs/sig-windows/generate-presubmits.sh.
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-aks-engine-windows-containerd-1-25
```
Comment (marosset) [inline config/jobs/kubernetes-sigs/sig-windows/release-1.25-windows-presubmits.yaml:4]: We should remove this entry (and any other entry for aks-engine)

Comment (nawazkh) [reply config/jobs/kubernetes-sigs/sig-windows/release-1.25-windows-presubmits.yaml]: addressed via commit : 3902356dd84622b0000f1b7c4ac5893017824c4e

**Pull request merged**